Core/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitor.cs

```diff
@@ -625,7 +630,7 @@ private bool IsCurrentExpressionEvaluatable (Expression expression)
     }
 #endif
 
-    private bool IsQueryableExpression (Expression expression)
+    private static bool IsQueryableExpression (Expression expression)
```

Comment:
Same as above regarding static. If there is an actual benefit to using static vs. instance, I think a separate dedicated PR would be better. In the code base, we rarely (in theory: never) use static unless the method needs to be static so if static has a measurable benefit for EF Core, there's probably more code in the call chain that needs to be changed to static.